server/index.ts

```diff
@@ -83,6 +83,15 @@ async function start(id: number, disconnect: () => void) {
   // Apply default rate limit to all routes
   app.use(defaultRateLimiter());
 
+  app.context.redirectOnClient = function (url: string) {
```

Comment:
```suggestion
  /** Perform a redirect on the browser so that the users auth cookies are included in the request. */
  app.context.redirectOnClient = function (url: string) {
```